sdk/core/core-tracing/src/utils/cache.ts

```diff
@@ -8,13 +8,14 @@ import { getGlobalObject } from "./global";
 // V2 = OpenTelemetry 0.2
 // V3 = OpenTelemetry 0.6.1
 // V4 = OpenTelemetry 1.0.0-rc.0
-const GLOBAL_TRACER_VERSION = 4;
+// V5 onwards will match OpenTelemetry version
+export const GLOBAL_TRACER_VERSION = "0.20.0";
 // preview5 shipped with @azure/core-tracing.tracerCache
 // and didn't have smart detection for collisions
-const GLOBAL_TRACER_SYMBOL = Symbol.for("@azure/core-tracing.tracerCache3");
+const GLOBAL_TRACER_SYMBOL = Symbol.for(`@azure/core-tracing.tracerCache${GLOBAL_TRACER_VERSION}`);
```

Comment:
If you're putting the version into the symbol name, you might as well delete the `else` clause on line 32, because we're never going to warn about incompatible global tracers. You can also remove `GLOBAL_TRACER_VERSION` entirely and delete `version` from the cache interface, since it's superfluous.

The point of the version was to track if the tracer we were storing was from a different and incompatible version of `core-tracing`.